PayPalNative/src/androidTest/java/com/braintreepayments/api/ExampleInstrumentedTest.java

```diff
@@ -15,7 +15,7 @@
  *
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
-@RunWith(AndroidJUnit4.class)
+@RunWith(RobolectricTestRunner.class)
 public class ExampleInstrumentedTest {
```

Comment:
I think this entire file can be deleted (it looks like we accidentally included it when creating this new module).